plugin/documents.py

```diff
@@ -483,8 +483,17 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                command = {}
+                for commands in responses.values():
+                    command = commands[0]
+                title = command.get('title')
+
+            code_actions_link = make_command_link('lsp_code_actions', title, {
+                "commands_by_config": responses
+            })
```

Comment:
There are some type issues here. Can try:
```suggestion
                title = ''
                for actions in responses.values():
                    title = actions[0].get('title')
            code_actions_link = make_command_link('lsp_code_actions', title, {"actions_by_config": responses})
```
I'd not call the variable "commands" because those are either `CodeAction`s or `Command`s so calling it "commands" is confusing.

Reply:
> There are some type issues here. Can try:
> 
> I'd not call the variable "commands" because those are either `CodeAction`s or `Command`s so calling it "commands" is confusing.

Thank you for your help, modified in f477b9c